timesketch/models/sigma.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module implements the sigma model."""
+
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import Integer
+from sqlalchemy import Unicode
+from sqlalchemy import UnicodeText
+
+from timesketch.models import BaseModel
+from timesketch.models.annotations import LabelMixin
+from timesketch.models.annotations import CommentMixin
+from timesketch.models.annotations import StatusMixin
+
+from timesketch.models.annotations import GenericAttributeMixin
+
+
+class Sigma(
+    BaseModel, LabelMixin, StatusMixin, CommentMixin, GenericAttributeMixin
+):
+    """Implements the Sigma model."""
```

Comment:
As discussed offline: Lets add `title` and `description` here as well to enable server side filtering of rules.

Reply:
done